src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -134,16 +138,28 @@ public static AddressBook getTypicalAddressBook() {
                 assert false : "not possible";
             }
         }
+
+        for (ReadOnlyPerson person : getTypicalWhitelistedPersons()) {
+            try {
+                ab.addWhitelistedPerson(person);
+            } catch (DuplicatePersonException e) {
+                assert false : "not possible";
```

Comment:
I've realised your assertion messages are very general for all assertion failures. Try to make them more specific so that you know what is the cause behind the failure

Reply:
Noted.